linux/pantos-service-node-server.service

```diff
@@ -7,7 +7,7 @@ StartLimitIntervalSec=0
 Type=forking
 Restart=always
 RestartSec=1
-ExecStart=/usr/local/bin/pantos-service-node-server
+ExecStart=runuser -u pantos -- bash -c "source /opt/pantos/service-node/virtual-environment/bin/activate; ./usr/local/bin/pantos-service-node-server"
```

Comment:
We should either activate the virtual environment from within the `pantos-service-node-server` script or move the script to `/opt/pantos/service-node`. Having it on the `PATH` in `/usr/local/bin` implies that it can be executed as-is.

Reply:
Good point. I moved it to `/opt/pantos/service-node`.